code/datums/status_effects/debuffs.dm

```diff
@@ -57,7 +65,7 @@
 
 /datum/status_effect/incapacitating/sleeping/tick()
 	if(owner.staminaloss)
-		owner.adjustStaminaLoss(-0.35) //reduce stamina loss by 0.35 per tick, 7 per 2 seconds
```

Comment:
Why not make sleeping do more since that disables you more than a normal stun so it's still the desired way to fix stamina rapidly